apps/transport/lib/transport_web/views/error_view.ex

```diff
@@ -9,7 +9,7 @@ defmodule TransportWeb.ErrorView do
     render(__MODULE__, "internal_server_error.html", assigns)
   end
 
-  def render("404.html", _assigns) do
-    render(__MODULE__, "internal_server_error.html", %{reason: dgettext("errors", "404: Page not available")})
```

Comment:
can't you use the 404.html?

Reply:
I cannot find the 404.html template. Did you mean I need to rename `not_found_error.html` to `404.html` ?

Reply:
nop, I don't know what I meant anymore...